cli/src/eval.rs

```diff
@@ -0,0 +1,40 @@
+use nickel_lang_core::{eval::cache::lazy::CBNCache, program::Program};
+
+use crate::{
+    cli::GlobalOptions,
+    error::{CliResult, ResultErrorExt},
+};
+
+#[derive(clap::Parser, Debug)]
+pub struct EvalCommand {}
+
+impl EvalCommand {
+    pub fn run(self, global: GlobalOptions) -> CliResult<()> {
+        let mut program = prepare(&global)?;
+        program
+            .eval_full()
+            .map(|t| println!("{t}"))
+            .report_with_program(program)
+    }
+
+    pub fn prepare(&self, global: &GlobalOptions) -> CliResult<Program<CBNCache>> {
+        prepare(global)
```

Comment:
I imagine `prepare` below isn't inlined here because it's used somewhere else outside of an `EvalCommand` value?